addons/mass_mailing/tests/common.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+
+
+class TestMassMailingCommon(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
it's classmethod. 
see this for more details : https://github.com/odoo/odoo/blob/master/addons/mail/tests/common.py#L28